go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/Microsoft/go-winio v0.4.16 // indirect
 	github.com/StackExchange/wmi v0.0.0-20190523213315-cbe66965904d // indirect
 	github.com/casbin/casbin v1.9.1
-	github.com/containerd/containerd v1.4.1-0.20201117152358-0edc412565dc // indirect
+	github.com/containerd/containerd v1.4.8 // indirect
```

Comment:
LGTM , In comments good to mention on what this updated containerd does to metigate vulnerabilities